### ISSUE TITLE
transform=ccrs.PlateCarree() now required when making maps

### ISSUE THREAD

**Issue by bradyrx** [lukelbd/proplot#58]
I think after 77f2b71b4927f9aa8b7024bbca53b87239203f2f, this issue arose.

Previously, one could do something like:

```python
import numpy as np
import proplot as plot
% import cartopy.crs as ccrs

data = np.random.rand(180, 360)
lats = np.linspace(-89.5, 89.5, 180)
lons = np.linspace(-179.5, 179.5, 360)

f, ax = plot.subplots(proj='robin')

ax.pcolormesh(lons, lats, data)
```

and you'd get the map. Now it's showing up blank and requires `ax.pcolormesh(lons, lats, data, transform=ccrs.PlateCarree())`. Maybe minor but a nice thing to not have to worry about with `proplot`.

**Comment by lukelbd** [comment]
...sorry, that was a dumb bug. Fixed by 05c38981.

**Comment by bradyrx** [comment]
FYI @lukelbd , `globe=True` doesn't work anymore for me with `contourf` after this (or maybe before this? I'm not sure). But it fixes the seam on `pcolormesh` but not `contourf` right now. I had to manually run the cartopy `add_cyclic_point`. 

I'm not sure how to reproduce this with simple data. It's always a mystery to me which data makes a seam.
